python-frontend/src/test/java/org/sonar/python/semantic/ProjectLevelSymbolTableTest.java

```diff
@@ -521,7 +521,7 @@ void importedStubModules() {
       """);
     ProjectLevelSymbolTable projectLevelSymbolTable = ProjectLevelSymbolTable.empty();
     projectLevelSymbolTable.addModule(tree, "my_package", pythonFile("mod.py"));
-    assertThat(projectLevelSymbolTable.typeShedDescriptorsProvider().stubModules()).containsExactlyInAnyOrder("math", "os");
+    assertThat(projectLevelSymbolTable.typeShedDescriptorsProvider().stubModules()).containsExactlyInAnyOrder("math", "os", "django", "django.urls.conf", "django.urls");
```

Comment:
That change comes from the fact that we always resolve those FQNs?

Reply:
Yes!